apps/style/common.scss

```diff
@@ -1201,6 +1201,22 @@ $workspace-header-button-margin: ($workspace-headers-height - $workspace-header-
   }
 }
 
+.show-code {
+  summary {
+    font-size: 18px;
+    line-height: 20px;
+    color: #5b6770;
+    font-family: "Gotham 5r", sans-serif;
+    font-weight: normal;
+    outline: none;
+    padding: 5px;
+  }
+
+  .generated-code-container {
+    margin: 25px 10px 25px -80px;
+  }
+}
+
```

Comment:
One option would be to pass in isMinecraft as a prop and style it differently depending on the value. Another would be to have a default set of styles, and allow the caller to pass in override styles.

If this is going to be too big a PITA, we can leave as is, but I really would like to get away from having styles living externally to the components. It makes it a lot harder to track down what the style is.